x-pack/plugins/security_solution/public/detection_engine/rule_exceptions/logic/use_exception_flyout_data.tsx

```diff
@@ -97,20 +101,38 @@ export const useFetchIndexPatterns = (rules: Rule[] | null): ReturnUseFetchExcep
       if (activeSpaceId !== '' && memoDataViewId) {
         setDataViewLoading(true);
         const dv = await data.dataViews.get(memoDataViewId);
-        const fieldsWithUnmappedInfo = await data.dataViews.getFieldsForIndexPattern(dv, {
-          pattern: '',
-          includeUnmapped: true,
-        });
+        const fields = dv.fields.map((field) => field.toSpec());
```

Comment:
This is duplication of the code above [`x-pack/plugins/security_solution/public/common/containers/source/index.tsx`](https://github.com/elastic/kibana/pull/158371/files#diff-2b9142306df119b8c31e8a66c9b93b2a83f74239b1655919881165ac06398623R154)

It is refactored in https://github.com/elastic/kibana/pull/155900. Left duplication to avoid unnecessary conflicts when we gonna merge that PR, since conflict description code is copied from there.